src/main/content/about.html

```diff
@@ -16,7 +16,7 @@
             <div class="col-sm-8">
                 <h3 id="intro_subtitle_1">The most flexible server runtime in the cosmos is open to Earth's Java<sup>TM</sup> developers.</h3>
                 <p id="intro_paragraph_1">
-                    Open Liberty provides developers with proven Java EE 7 technology and the latest Eclipse <a href="https://microprofile.io" target="new" class="light_green_link_dark_background">MicroProfile<sup>TM</sup></sup></a> capabilities for building
+                    Open Liberty provides developers with proven Java EE 7 technology and the latest Eclipse <a href="https://microprofile.io" target="new" class="light_green_link_dark_background">MicroProfile&reg;</a> capabilities for building
```

Comment:
I'd put the link around the Eclipse MicroProfile rather than just Eclipse.

Reply:
Fixed